kibana/Dockerfile

```diff
@@ -1,55 +1,44 @@
-FROM rhscl/nodejs-6-rhel7
+FROM rhoar-nodejs/nodejs-10
 
 MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 EXPOSE 5601
 
-ENV ELASTICSEARCH_URL=https://logging-es:9200 \
+ENV ELASTICSEARCH_URL=https://elasticsearch:9200 \
```

Comment:
Do we need the full URL so global proxy works correctly?

Reply:
i dont think we need to worry about that, but it couldn't hurt.. will update